### ISSUE TITLE
EXPLODE returns null values in 0.10.0

### ISSUE THREAD

**Issue by mikebin** [confluentinc/ksql#5723]
**Describe the bug**
The `EXPLODE` function is returning `null` values in the 0.10.0 release.

**To Reproduce**
ksqlDB version: 0.10.0

```sql
create stream arraytest (val array<int>) with (kafka_topic='arraytest', value_format='json', partitions=1);

insert into arraytest (val) values (array[1,2,3,4,5]);

select explode(val) from arraytest emit changes;
```

**Expected behavior**
```+--------------------+
|KSQL_COL_0          |
+--------------------+
|1                |
|2                |
|3                |
|4                |
|5                |
```

**Actual behaviour**
```+--------------------+
|KSQL_COL_0          |
+--------------------+
|null                |
|null                |
|null                |
|null                |
|null                |
```

**Comment by purplefox** [comment]
Probably related to recent key field changes I would guess.

**Comment by big-andy-coates** [comment]
Yeah, its related to recent key changes.  Worth noting this is not a regression, but an edge-case with a new feature. The issue only arises with UDTFs on key-less streams, which were only introduced with 0.10.

Should definitely be fixed in 0.10, but shouldn't be a blocker for 0.10.1 release.
